src/components/Proposal/Create/SchemeForms/CreateSchemeRegistrarProposal.tsx

```diff
@@ -138,12 +138,18 @@ class CreateSchemeRegistrarProposal extends React.Component<IProps, IState> {
     } else {
       proposalType = IProposalType.SchemeRegistrarEdit;
     }
+
+    let permissionString = "0x" + permissions.toString(16).padStart(8, "0");
```

Comment:
Not a big deal, but I prefer an if/else or switch/case/default pattern here, to save an unnecessary assignment to `permissionString` when schemeName === "Plugin Manager"

Reply:
Can you please add a comment in the code explaining why the special case hack is required?  Explaining why `permissions` is good for every scheme except "Plugin Manager".  This should not be the case, seems like a bug elsewhere.  Need to explain in the code.